src/main/scala/com/avast/clients/stor/StorClient.scala

```diff
@@ -0,0 +1,13 @@
+package com.avast.clients.stor
+
+import java.io.InputStream
+
+import better.files.File
+
+trait StorClient[F[_]] {
+  def head(sha256: String): F[Either[StorException, HeadResult]]
```

Comment:
Sha256 will be heavily used so I would prefer to create a new case class that represents `Sha256`.

Reply:
Fixed